<launch>
    <!-- 标定点1在基坐标系下的位置 -->
    <param name="marker1_x" value="8.2296" type="double" />
    <param name="marker1_y" value="0" type="double" />
    <!-- 标定点2在基坐标系下的位置 -->
    <param name="marker2_x" value="30.1752" type="double" />
    <param name="marker2_y" value="0" type="double" />
    <!-- 雷达在基坐标系下的位置和yaw轴偏角(角度制) -->
    <param name="lidar_x" value="20" type="double" />
    <param name="lidar_y" value="2.5" type="double" />
    <param name="lidar_yaw" value="275" type="double" /> 
    <!-- 是否在实机上运行 -->
    <param name="in_real_machine" value="true" type="bool" />

    <!-- 
    2-11 18点到19点参数
    <param name="lidar_x" value="20" type="double" />
    <param name="lidar_y" value="2.5" type="double" />
    <param name="lidar_yaw" value="275" type="double" /> 
    -->

    <!-- 
    2-11 22点到0点参数 -s 85 y=2.4
    <param name="lidar_x" value="15" type="double" />
    <param name="lidar_y" value="1.5" type="double" />
    <param name="lidar_yaw" value="-72" type="double" />
    -->

    <node pkg="curling_detect" type="prompt_graph" name="prompt_graph" output="screen" />
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find curling_detect)/config/prompt_graph.rviz" required="true"/>

</launch>



